larq/quantizers.py

```diff
@@ -164,6 +164,10 @@ def call(self, inputs):
         if hasattr(self, "flip_ratio"):
             self.add_metric(self.flip_ratio(inputs))
 
+    @property
+    def non_trainable_weights(self):
+        return []
```

Comment:
We'll need to be careful with that if we add quantizers that actually rely on `add_weights` in the future, but for now it's fine since this is not a public API.